src/Apps/Order/Utils/getStatusCopy.tsx

```diff
@@ -71,13 +76,70 @@ export const getStatusCopy = (order, logger?): StatusPageConfig => {
       }
     case "PROCESSING_APPROVAL":
       return {
-        title: `${approvedTitle(isOfferFlow)}. Payment processing.`,
-        description: (
+        title: `${processingApprovalTitle(isOfferFlow, isWireTransfer)}`,
+        description: isWireTransfer ? null : (
           <>
             Thank you for your purchase. {deliverText(order)}More delivery
             information will be available once your order ships.
           </>
         ),
+        alertMessageTitle: isWireTransfer
+          ? "Please proceed with the wire transfer to complete your purchase"
+          : null,
+        alertMessage: isWireTransfer ? (
+          <>
+            <Text>
+              Please provide your proof of payment within 7 days. After this
+              period, your order will be eligible for cancellation by the
+              gallery.
+            </Text>
+            <Spacer mt={30} />
+            <Text>
+              1. &nbsp; Find the order total and Artsy's banking details below.
+            </Text>
+            <Text>
+              2. &nbsp;Please inform your bank that you will be responsible for
+              all wire transfer fees.
+            </Text>
+            <Text>
+              3. &nbsp;Once you have made the transfer, please email
+              orders@artsy.net with your proof of payment.
+            </Text>
+          </>
+        ) : null,
+        content: isWireTransfer ? (
+          <>
+            <Text
+              variant={["xs", "sm-display"]}
+              fontWeight="bold"
+              color="black100"
+            >
+              Send wire transfer to
+            </Text>
+            <Spacer mt={10} />
```

Comment:
I know it's weird that we use numeric keys but, use the token values `1`, `2`, etc.